packages/core/src/browser/fetchObservable.ts

```diff
@@ -52,7 +52,7 @@ function createFetchObservable() {
 
           const context = callMonitored(beforeSend, null, [observable, input, init])
           if (context) {
-            responsePromise = originalFetch.call(this, context.input, context.init)
+            responsePromise = originalFetch.call(this, context.input as Request | string, context.init)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we could use the existing type
```suggestion
            responsePromise = originalFetch.call(this, context.input as RequestInfo, context.init)
```